dev/integration_tests/new_gallery/lib/studies/shrine/supplemental/desktop_product_columns.dart

```diff
@@ -60,10 +60,10 @@ class DesktopProductCardColumn extends StatelessWidget {
                 card = DesktopProductCard(
                   product: products[productCardIndex],
                   imageWidth: startLarge
-                      ? ((productCardIndex.isEven)
+                      ? (productCardIndex.isEven
                           ? largeImageWidth
                           : smallImageWidth)
-                      : ((productCardIndex.isEven)
+                      : (productCardIndex.isEven
                           ? smallImageWidth
                           : largeImageWidth),
```

Comment:
If we're in the process of cleaning this up, we might as well simplify it further:

```dart
                  imageWidth: startLarge == productCardIndex.isEven
                      ? largeImageWidth
                      : smallImageWidth,
```